libs/vectorise/include/vectorise/fixed_point/type_traits.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include <type_traits>
+
+#include "core/byte_array/byte_array.hpp"
+#include "vectorise/fixed_point/fixed_point.hpp"
+
+namespace fetch {
+namespace math {
```

Comment:
should probably change the namespace to fetch::vectorise::meta, or fetch::vectorise::fixed_point::meta or something like that

Reply:
switched to fetch::fixed_point::meta

Reply:
reverted to fetch::math::meta for (in)consistency reasons, ie fetch::fixed_point::meta::IsFixedPoint but fetch::math::meta::IsFloat.